### ISSUE TITLE
Thumbnail images are missing alt text

### ISSUE THREAD

**Issue by srappel** [geoblacklight/geoblacklight_sidecar_images#44]
Thumbnail images do not include an alt text attribute when a thumbnail is present. Doesn't appear to be an issue with the SVG icons used for datasets with no thumbnail image.

<img width="495" height="444" alt="Image" src="https://github.com/user-attachments/assets/75dedc4d-e7cd-4c01-8e38-3e719ff2a73f" />

Best practice may be to include alt="" or an aria presentation tag to encourage screen readers to skip the content.

**Comment by ewlarson** [comment]
Good catch! I'm unlikely to get around to this anytime soon. If you can submit a PR, I will review and merge tho!

**Comment by ewlarson** [comment]
Closing via: https://github.com/geoblacklight/geoblacklight_sidecar_images/pull/45
